fetch-utils.js

```diff
@@ -1,8 +1,73 @@
-const SUPABASE_URL = '';
-const SUPABASE_KEY = '';
+const SUPABASE_KEY = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYzOTUwODU3MCwiZXhwIjoxOTU1MDg0NTcwfQ.b9_dCGIQkWfhzS3QZihLzQkD3n-sAt3L9swaEU7JzqI';
+const SUPABASE_URL = 'https://rfwnchvtfqbachqhdfbi.supabase.co';
 
 const client = supabase.createClient(SUPABASE_URL, SUPABASE_KEY);
 
+export async function createCity(city) {
```

Comment:
city is defined but never used error from below is because you have defined city as an argument in your createCity function, but never actually use that variable anywhere.  I believe you would want to use it in your insert call instead of the string 'city'.